rush.json

```diff
@@ -346,8 +346,8 @@
       "projectFolder": "sdk/eventhub/event-processor-host"
     },
     {
-      "packageName": "@azure/keyvault",
```

Comment:
Are we intentionally removing the other keyvault library from our workspace here?

Reply:
@bsiegel if I leave it the build fails somehow. Also, I believe we're indeed removing it, based on a chat I had yesterday with @jonathandturner . Please confirm ✨

Reply:
/cc @schaabs